demo/demo_notebooks/fraud_detection.ipynb

```diff
@@ -30,7 +30,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 1,
```

Comment:
I am not sure what these changes have to do with the current PR.

Reply:
It's an accident, I'll remove this file from the PR.